api/swagger.yaml

```diff
@@ -77,43 +90,337 @@ tags:
+
+      ### `cancelEvent`
+
+      Cancels one or more `Event`. Since events can only be created and not deleted nor updated, a `cancelEvent` is needed when you need to change
+      or remove an event. To be canceled events should be set as `involvedObjects`. Note that you can't cancel events in another lifecycle phase.
+
+      #### Applicable life cycle phases
+
+      | Planned | Projected | Actual | Realized |
+      | :-----: | :-------: | :----: | :------: |
+      |  ●      | ●         | ●      | ●        |
+
+      #### Applicable entities
+
+      | Location | Vehicle | Shipment | Trip  | Route | Actor | Constraint | Sensor | Event |
+      | :------: | :-----: | :------: | :---: | :---: | :---: | :--------: | :----: | :---: |
+      |          |         |          |       |       |       |            |        | ●     |
+
+      ### `cancelAllEntityEvents`
+
+      This event can be used to cancel _all_ events that are associated with a given entity. This might be needed e.g. in cases where a `Trip` with
+      lots of `planned` events is published and the planning is changed. In such a case, it can be more efficient to just cancel all events and start
+      over.
+
+      #### Applicable life cycle phases
+
+      | Planned | Projected | Actual | Realized |
+      | :-----: | :-------: | :----: | :------: |
+      |  ●      | ●         | ●      | ●        |
+
+      #### Applicable entities
+
+      | Location | Vehicle | Shipment | Trip  | Route | Actor | Constraint | Sensor | Event |
+      | :------: | :-----: | :------: | :---: | :---: | :---: | :--------: | :----: | :---: |
+      | ●        | ●       | ●        | ●     | ●     | ●     | ●          | ●      |       |
+
+      ### `cancelGroupedEvents`
+
+      This event can be used to cancel all events that share the same `groupingId`. The grouping IDs to be cancelled myst be set in the
+      `cancelledGroupingIds` variable, which can contain an array of groupingId's.
+
+      #### Applicable life cycle phases
+
+      | Planned | Projected | Actual | Realized |
+      | :-----: | :-------: | :----: | :------: |
+      |  ●      | ●         | ●      | ●        |
+
+      #### Applicable entities
+
+      | Location | Vehicle | Shipment | Trip  | Route | Actor | Constraint | Sensor | Event |
+      | :------: | :-----: | :------: | :---: | :---: | :---: | :--------: | :----: | :---: |
+      | ●        | ●       | ●        | ●     | ●     | ●     | ●          | ●      |       |
+
+      ### `ignoreEvent`
+
+      Ignore one or more `Event`. To be ignored event(s) should be set as `involvedObject(s)`. Only use `ignoreEvents` to mark events from other
+      systems or actors to be ignored if there is no other way to prevent the events from being sent at all.
+
+      #### Applicable life cycle phases
+
+      | Planned | Projected | Actual | Realized |
+      | :-----: | :-------: | :----: | :------: |
+      |         |           | ●      | ●        |
```

Comment:
Maybe add an explaination as to why only actual and realized lifecycles are applicable here?

Reply:
Like in `cancelEvent`, all lifecycles are applicable and one can only ignore events that are in the same lifecycle.